colossalai/graph/passes/adding_split_node_pass.py

```diff
@@ -0,0 +1,54 @@
+import torch
+
+from torch.fx import symbolic_trace
+from torch.fx.node import Node
+from torch.fx.passes.split_module import split_module
+
+
+def pipe_split():
+    pass
+
+
+def balanced_split_node(gm: torch.fx.GraphModule, pp_size: int):
+    mod_graph = gm.graph
+    total_param_amount = 0
+    for param in mod_graph.owning_module.parameters():
+        total_param_amount += param.numel()
```

Comment:
Will calling `.numel()` work with ColossalAI context?

Reply:
Yes, the param with ColoInit will be a ColoTensor, it has numel method and works as expected.